ng2-components/ng2-alfresco-upload/src/components/upload-drag-area.component.ts

```diff
@@ -54,6 +54,9 @@ export class UploadDragAreaComponent {
     @Input()
     rootFolderId: string = UploadDragAreaComponent.DEFAULT_ROOT_ID;
 
+    @Input()
+    uploadToECM: boolean = true;
```

Comment:
Please use the upload directive (it is inside the alfresco core component) instead of change the alfresco upload.